pmm-app/tests/pages/pmmInventoryPage.js

```diff
@@ -207,4 +208,33 @@ module.exports = {
 
     assert.equal(count, otherDetails, 'Check data!');
   },
+
+  async getCellValue(rowNumber, columnNumber) {
+    const value = await I.grabTextFrom(`//tr[@data-qa="table-row"][${rowNumber}]/td[${columnNumber}]`);
+
+    return value.toLowerCase();
+  },
+
+  async checkSort(columnNumber) {
+    I.waitForVisible(this.fields.tableRow, 20);
+    const rowCount = await I.grabNumberOfVisibleElements(this.fields.tableRow);
+    let tmp;
+
+    for (i = 0; i < rowCount; i++) {
+      const cellValue = await this.getCellValue(i + 1, columnNumber);
+
+      if (i == 0) {
+        // Do nothing for the first run
+        tmp = cellValue;
+      } else {
+        if (tmp.localeCompare(cellValue) === 1) {
+          assert.fail('The array is not sorted correctly from a-z');
```

Comment:
How is this assert message helpful when debuging a failure?

Reply:
It does the same as other `assert` functions in this repo : you will see this message in the test run console log when the test fails. Very helpful and very neat.

Reply:
Or did you mean , that we have to print both values when it fails?

Reply:
Done.